xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -828,8 +828,15 @@ void CRenderManager::FlipPage(volatile std::atomic_bool& bStop, double pts /* =
         presentmethod = PRESENT_METHOD_BOB;
         invert = true;
       }
+      else if (interlacemethod == VS_INTERLACEMETHOD_IMX_ADVMOTION_DOUBLE)
```

Comment:
@FernetMenta 

that was the idea, to reuse silly AutoInterlaceMethod to ask renderer as the call ends there ...
what is the way you have on your mind?